test/generators/scaffold_controller_generator_test.rb

```diff
@@ -1,4 +1,5 @@
 require 'test_helper'
+require 'generators/rails/resource_override'
```

Comment:
Yeah. When "rails generate resource ..." runs, Rails generator loading makes sure everything gets initialized and specifically calls that new `generator` block in the Railtie. But in the test case, only the Railtie initializers run, but not the `generator` block. So this wasn't needed before since the initializers were always loading all the generators.

It definitely works from the command line still.